lib/messages.js

```diff
@@ -379,12 +405,15 @@ const OplogMessage = {
         : OplogMessageV0.decode(state)
 
       const chk = m.checkpoint
+      const checkpoint = chk
+        ? { system: chk[0], encryption: null, user: chk.slice(1) }
```

Comment:
is encryption always null here? not sure i follow

Reply:
yeh this is v0 and v1 message so encryption view doesn't exist yet